.wp-env.json

```diff
@@ -0,0 +1,13 @@
+{
+	"plugins": [
+		"https://downloads.wordpress.org/plugin/woocommerce.latest-stable.zip",
+		"https://github.com/WP-API/Basic-Auth/archive/master.zip",
+		"."
+	],
+	"themes": [
+		"https://downloads.wordpress.org/theme/storefront.latest-stable.zip"
+	],
+	"mappings": {
+		".htaccess": "./bin/.htaccess"
+	}
```

Comment:
This is a workaround for the permissions issue with non default WP versions in builds. In those builds the `.htaccess` file isn't created because of broken permissions in the wp-cli container. By mapping an already prepared .htaccess file we avoid the need to write the file during the setup phase.